### PR TITLE
[REFACTOR] Refactor Worker module scala code style

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/DeviceMonitor.scala [hunk 1]
```diff
@@ -37,36 +37,47 @@ import com.aliyun.emr.rss.common.util.Utils._
...
 
   class ObservedDevice(val deviceInfo: DeviceInfo) {
-    var mountInfos: ListBuffer[MountInfo] = deviceInfo.mountInfos
+    var mounts: ListBuffer[MountInfo] = deviceInfo.mounts
     val observers: ConcurrentSet[DeviceObserver] = new ConcurrentSet[DeviceObserver]()
 
     val sysBlockDir = RssConf.sysBlockDir(essConf)
     val statFile = new File(s"$sysBlockDir/${deviceInfo.name}/stat")
-    val inFlightFile = new File(s"$sysBlockDir/${deviceInfo.name}/inflight")
+    val inFlightFile = new File(s"$sysBlockDir/${deviceInfo.name}/in-flight")
```
Comment (FMX) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/DeviceMonitor.scala:72]: The word "Inflight" is a filename.

Comment (AngersZhuuuu) [reply server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/DeviceMonitor.scala]: Revert this change

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala [hunk 2]
```diff
@@ -356,9 +299,9 @@ private[deploy] class Worker(
     // reserve success, update status
     partitionLocationInfo.addMasterPartitions(shuffleKey, masterPartitions)
     partitionLocationInfo.addSlavePartitions(shuffleKey, slavePartitions)
-    workerInfo.allocateSlots(shuffleKey, masterPartitions.size() + slavePartitions.size())
-    logInfo(s"Reserved ${masterPartitions.size()} master location and ${slavePartitions.size()}" +
-      s" slave location for $shuffleKey master: ${masterPartitions}\nslave: ${slavePartitions}.")
+    workerInfo.allocateSlots(shuffleKey, masterPartitions.size + slavePartitions.size)
+    logInfo(s"Reserved ${masterPartitions.size} master location and ${slavePartitions.size}" +
+      s" slave location for $shuffleKey master: $masterPartitions\nslave: $slavePartitions.")
```
Comment (AngersZhuuuu) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala:304]: remove unnecessary parentheses

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/WorkerSource.scala [hunk 3]
```diff
@@ -20,67 +20,70 @@ package com.aliyun.emr.rss.service.deploy.worker
...
-
-  val FlushDataTime = "FlushDataTime"
+  val SERVLET_PATH = "/metrics/prometheus"
 
-  val OpenStreamTime = "OpenStreamTime"
+  val COMMIT_FILE_TIME = "CommitFilesTime"
+  val RESERVE_SLOTS_TIME = "ReserveSlotsTime"
+  val FLUSH_DATA_TIME = "FlushDataTime"
+  val OPEN_STREAM_TIME = "OpenStreamTime"
+  val FETCH_CHUNK_TIME = "FetchChunkTime"
```
Comment (AngersZhuuuu) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/WorkerSource.scala:61]: Use uppercase for constants

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala [hunk 4]
```diff
@@ -955,9 +897,45 @@ private[deploy] class Worker(
...
+  def isRegistered: Boolean = {
     registered.get()
   }
+
+  private def createTransportServer(
+      conf: RssConf,
+      module: String,
+      port: Int,
+      defaultIOThreads: Int,
+      isPushDataServe: Boolean): TransportServer = {
```
Comment (waitinfuture) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala:909]: Maybe we should rename to needsLimiter

File: server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/LocalStorageManager.scala [hunk 5]
```diff
@@ -187,33 +188,31 @@ private[worker] final class LocalStorageManager(
...
+      cleaners.put(dir,
         ThreadUtils.newDaemonCachedThreadPool(s"Disk-cleaner-${dir.getAbsoluteFile}", 1))
     }
     cleaners
   }
 
   private val deviceMonitor = DeviceMonitor.createDeviceMonitor(conf, this, workingDirsSnapshot())
 
-  private val diskFlushers: ConcurrentHashMap[File, DiskFlusher] = {
+  private val diskFlusher: ConcurrentHashMap[File, DiskFlusher] = {
```
Comment (waitinfuture) [inline server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/LocalStorageManager.scala:210]: I think better to use diskFlushers

**Pull request closed without merging**